index.bs

```diff
@@ -4827,14 +4827,20 @@ operation. Likewise, clients can choose to produce a [=client extension output=]
 understand by encoding the [=authenticator extension output=] value into JSON, provided that the CBOR output uses only types
 present in JSON.
 
-When clients choose to pass through extensions they do not recognize,
+When [=clients=] choose to pass through extensions they do not recognize,
 the JavaScript values in the [=client extension inputs=]
 are converted to [=CBOR=] values in the [=authenticator extension inputs=].
 When the JavaScript value is an [=%ArrayBuffer%=], it is converted to a [=CBOR=] byte array.
 When the JavaScript value is a non-integer number, it is converted to a 64-bit CBOR floating point number.
 Otherwise, when the JavaScript type corresponds to a JSON type, the conversion is done
 using the rules defined in Section 4.2 of [[!RFC7049]] (Converting from JSON to CBOR),
 but operating on inputs of JavaScript type values rather than inputs of JSON type values.
+Note that the numeric conversion rules have the consequence that
+when a client passes through an extension it does not recognize,
+if the extension uses floating point values,
+[=authenticators=] need to be prepared to receive those values as [=CBOR=] integers
+when the floating point values used happen to be integers,
+should the [=authenticator=] want the extension to always work without actual [=client=] support for it.
```

Comment:
This is all one long sentence -- suggested rewrite (plus I'd move this Note to under line 4834 where numeric conversion is discussed):

Note: Due to Javascript's numeric conversion rules, any floating point values may be converted to integers. Thus, [=authenticators=] need to be prepared to receive those values as either [=CBOR=] floating-point or as [=CBOR=] integers. If this is not feasible, the extension may be designed with the [=client=] performing any necessary value type conversions.